app/server/utils/user.py

```diff
@@ -564,3 +564,29 @@ def proccess_create_or_modify_user_request(attribute_dict,
     }
 
     return response_object, 200
+
+
+def send_onboarding_message(to_phone, first_name, credits, one_time_code):
```

Comment:
moved out from phone.py. the other ones didn't seem to be used at all so i just deleted them